static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -450,7 +456,10 @@ const SPECIAL_FUNCTIONS: SpecialFunctions = {
     const uniqueUsers = data.count_unique_user;
     const userMisery = data[userMiseryField];
 
-    const miseryLimit = parseInt(userMiseryField.split('_').pop() || '', 10) || undefined;
+    let miseryLimit = parseInt(userMiseryField.split('_').pop() || '', 10);
+    if (isNaN(miseryLimit)) {
```

Comment:
What does userMiseryField become that this `NaN` check succeeds and we switch to using projectThresholdConfig below?

Reply:
Good question, `userMiseryField` can either be `user_misery(X00)` or `user_misery_new()` which will use the threshold values configured on the project level, so miseryLimit may be NaN